documentation/modules/proc-creating-configmap.adoc

```diff
@@ -0,0 +1,71 @@
+// Module included in the following assemblies:
+//
+// assembly-external-logging.adoc
+
+[id='creating-configmap_{context}']
+= Creating a ConfigMap for logging
+
+To use a ConfigMap to define logging properties, you create the ConfigMap and then reference it as part of the logging definition in the `spec` of a resource.
+
+The ConfigMap must contain the appropriate logging configuration.
+
+* `log4j.properties` for Kafka components and the Kafka Bridge
+* `log4j2.properties` for the Topic Operator and User Operator
+
+The configuration must be placed under these properties.
+
+Here we demonstrate how a ConfigMap defines a root logger for a Kafka resource.
+
+.Procedure
+
+. Create the ConfigMap.
++
+You can create the ConfigMap as a YAML file or from a properties file at the `kubectl` command line.
++
+ConfgMap example with a root logger definition for Kafka:
++
+[source,yaml,subs="+attributes"]
+----
+kind: ConfigMap
+apiVersion: {KafkaApiVersion}
+metadata:
+  name: logging-configmap
+data:
+  log4j.properties:
+  # logger settings for resource
```

Comment:
I think this comment is a bit confusing. Is it part of the log4j.properties file in which case it is misaligned? Or is it part of the YAML?

Reply:
I removed it.